app/javascript/components/student/editor/TestRunSummary.tsx

```diff
@@ -3,16 +3,16 @@ import { Submission, TestRun, TestRunStatus } from '../Editor'
 import { TestRunChannel } from '../../../channels/testRunChannel'
 import { TestRunSummaryContent } from './TestRunSummaryContent'
 import { fetchJSON } from '../../../utils/fetch-json'
+import { typecheck } from '../../../utils/typecheck'
+import { camelizeKeys } from 'humps'
 
 export function TestRunSummary({
-  testRun,
+  submission,
```

Comment:
I'm not clear on why you've changed this in this file. It feels like we've more tightly coupled things, rather than losening the coupling. Can you explain?

Reply:
It looks like the reason for this change is the use in several places of `submission.links` and `submission.uuid`. So instead of passing in the test run and the submission properties separately, it provides just the submission. Is that correct?

Reply:
Yes. It depends on a lot of things on submission that's why I just passed submission down.

Reply:
I've updated the PR and I've moved things back to how they were before. Agree with making the coupling less loose.